mozilla_django_oidc/auth.py

```diff
@@ -89,6 +89,7 @@ def authenticate(self, **kwargs):
         code = kwargs.pop('code', None)
         state = kwargs.pop('state', None)
         nonce = kwargs.pop('nonce', None)
+        session = kwargs.pop('session', None)
```

Comment:
please pass here the session_key and use `Session.objects.get(key)` for the session.